test/manual/WebPageTranscoderOptions.html

```diff
@@ -0,0 +1,21 @@
+<script src="../../bin/traceur.js"></script>
+<script src="../../src/bootstrap.js"></script>
+<script>
+  new traceur.WebPageTranscoder(document.location.href, {
+    asyncFunctions: true
+  }).run();
+</script>
+<script type="module">
+var async;
```

Comment:
This test is pretty silly since it does not even test async, just that async works as an identifier even when we use asyncFunctions.